cmd/yggd/worker.go

```diff
@@ -0,0 +1,270 @@
+package main
+
+import (
+	"bufio"
+	"fmt"
+	"io"
+	"io/ioutil"
+	"os"
+	"path/filepath"
+	"regexp"
+	"strconv"
+	"strings"
+	"time"
+
+	"git.sr.ht/~spc/go-log"
+	"github.com/pelletier/go-toml"
+	"github.com/redhatinsights/yggdrasil"
+	"github.com/rjeczalik/notify"
+	"golang.org/x/net/http/httpproxy"
```

Comment:
Could you please tidy up the go.mod file as the golang.org/x/net is becoming a
direct dependency now?